packages/salesforce-adapter/src/adapter.ts

```diff
@@ -479,6 +486,9 @@ export const salesforceAdapterResolveValues: ResolveValuesFunc = async (
     : resolvedElement
 }
 
+export const resolveChanges = (changes: readonly Change[], getLookupNameFunc: GetLookupNameFunc): Promise<Change[]> =>
```

Comment:
```suggestion
export const resolveSalesforceChanges = (changes: readonly Change[], getLookupNameFunc: GetLookupNameFunc): Promise<Change[]> =>
```
nit: As `resolveChanges` might be a too generic export that may lead to wrong auto imports